lib/screens/admin_order_detail_page.dart

```diff
@@ -0,0 +1,585 @@
+        placeholder: kTransparentImage,
+        image: product.imageUrls.first,
+      );
+    }
+
+    return ClipRRect(
+      borderRadius: BorderRadius.circular(12.0),
+      child: productImage,
+    );
+  }
+
+  Widget _buildOrderItemTextInfo(OrderItem orderItem, Product product) {
+    return Container(
+      height: 75,
+      padding: EdgeInsets.only( right: 50.0 ),
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: <Widget>[
+          _buildProductName(product),
+          _buildOrderItemColorSize(orderItem, product),
+          _buildOrderItemPrice(orderItem, product),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildProductName(Product product) {
+    return Container(
+      child: Text(
+        product.name,
+        style: TextStyle(
+          fontSize: 16,
+          fontWeight: FontWeight.w600,
+          color: paletteBlackColor,
+        ),
+      ),
+    );
+  }
+
+  Widget _buildOrderItemColorSize(OrderItem orderItem, Product product) {
+    return Container(
+      child: Text(
+        // orderItem.color.toString() + '/' + orderItem.size.toString(),
+        '?サイズ/なにいろ',
+        style: TextStyle(
+          fontSize: 14,
+          fontWeight: FontWeight.w300,
+          color: paletteBlackColor,
+        ),
+      ),
+    );
+  }
+
+  Widget _buildOrderItemPrice(OrderItem orderItem, Product product) {
+    return Container(
+      child: Text(
+        "¥" + NumberFormat("#,##0").format(orderItem.price),
+        style: TextStyle(
+          fontSize: 16,
+          fontWeight: FontWeight.w600,
+          color: paletteBlackColor,
+        ),
+      ),
+    );
+  }
+
+  Widget _buildOrderItemQuantity(OrderItem orderItem, Product product) {
+    return Container(
+      height: 85,
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.end,
+        children: <Widget>[
+          Row(
+            children: <Widget>[
+              Container(
+                color: kPaletteWhite,
+                height: 30,
+                width: 40,
+                alignment: Alignment.center,
+                child: Text(
+                  orderItem.quantity.toString(),
+                  style: TextStyle(
+                    fontSize: 16,
+                    fontWeight: FontWeight.bold,
+                    color: paletteBlackColor,
+                  ),
+                ),
+              ),
+              Text(
+                ' 点',
+                style: TextStyle(
+                  fontSize: 16,
+                  fontWeight: FontWeight.w300,
+                  color: paletteBlackColor,
+                ),
+              )
+            ],
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildOrderPriceDescriptionBar() {
+    return Container(
+      padding: EdgeInsets.symmetric( horizontal: 16.0 ),
+      height: 40,
+      child: Container(
+        color: kPaletteWhite,
```

Comment:
Maybe add a padding here as the text in this Container is literally stuck to the edges. Please consider adding this to the Container:

`padding: const EdgeInsets.symmetric(horizontal: 8.0),`